code/models/index.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const Sequelize = require('sequelize');
+const basename = path.basename(__filename);
+const env = process.env.NODE_ENV || 'development';
+const config = require(__dirname + '/../config/config.json')[env];
+const db = {};
+
+let sequelize;
+if (config.use_env_variable) {
+  sequelize = new Sequelize(process.env[config.use_env_variable], config);
+} else {
+  sequelize = new Sequelize(config.database, config.username, config.password, config);
+}
+
+fs
+  .readdirSync(__dirname)
+  .filter(file => {
+    return (file.indexOf('.') !== 0) && (file !== basename) && (file.slice(-3) === '.js');
+  })
+  .forEach(file => {
+    const model = require(path.join(__dirname, file))(sequelize, Sequelize.DataTypes);
+    db[model.name] = model;
+  });
+
+Object.keys(db).forEach(modelName => {
```

Comment:
It looks like we go through all the models we put into the `db` object and if it has an `associate` method, we can call it with `db`, the object we're in, because that passes the function in question the argument of `sequelize` that it needs as its first parameter? The function defined in `model` returns a connected up value of the data in object form ready to be used, so that becomes the returned value of each of these?

This is a bit fuzzy here for me so I'd love to walk through sometime on this specific file. Maybe I'll understand more after doing a tutorial in either GraphQL or Sequelize!